.circleci/config.yml

```
@@ -2,8 +2,8 @@
 # syntax, such as anchors, will be fixed automatically.
 version: 2.1
 orbs:
-  shared: getoutreach/shared@2.19.2
-  queue: eddiewebb/queue@1.8.4
+  shared: getoutreach/shared@2.21.0
```

suggestion: we may want to use the latest version
```suggestion
  shared: getoutreach/shared@2.24.0
```

---

Sounds good